stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -15,10 +15,133 @@ import SwiftPrivate
 import Darwin
 #elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
 import Glibc
+#elseif os(Windows)
+import MSVCRT
+import WinSDK
 #endif
```

Comment:
Wouldn't it make most sense to structure this like:

    #if canImport(Darwin)
    import Darwin
    #elseif os(Windows)
    import MSVCRT
    import WinSDK
    #elseif canImport(Glibc)
    import Glibc
    #endif

Then things work automatically for other POSIX platforms which probably use Glibc (NetBSD, OpenBSD, etc.?) without needing to explicitly update the list every time someone adds a new OS to Swift.

Reply:
This is why I had proposed having a `C` module which hides this mess.  In any case, I suppose that I can change that, but this pattern is prevalent through out the code base.  I think it makes sense for someone to go through and catch them all later.

Reply:
We've talked about such a module before and the primary problem is what to name it. :-) I think there'd be support on -evolution.